content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade-1-13.md

```diff
@@ -33,18 +33,38 @@ This page explains how to upgrade a Kubernetes cluster created with `kubeadm` fr
 
 {{% capture steps %}}
 
-## Upgrade the control plane
+## Determine what version you want to upgrade to
 
-1.  On your master node, upgrade kubeadm:
+1.  Find the latest stable 1.13 version:
 
-    {{< tabs name="k8s_install" >}}
+    {{< tabs name="k8s_install_versions" >}}
     {{% tab name="Ubuntu, Debian or HypriotOS" %}}
-    apt-mark unhold kubelet kubeadm && \
-    apt-get update && apt-get upgrade -y kubelet kubeadm && \
-    apt-mark hold kubelet kubeadm
+    apt update
+    apt-cache policy kubeadm
+    # find the latest 1.13 version in the list
+    # it should look like 1.13.x-00, where x is the latest patch
     {{% /tab %}}
     {{% tab name="CentOS, RHEL or Fedora" %}}
-    yum upgrade -y kubeadm --disableexcludes=kubernetes
+    yum list --showduplicates kubeadm --disableexcludes=kubernetes
+    # find the latest 1.13 version in the list
+    # it should look like 1.13.x-0, where x is the latest patch
+    {{% /tab %}}
+    {{< /tabs >}}
+
+## Upgrade the control plane node
+
+1.  On your control plane node, upgrade kubeadm:
+
+    {{< tabs name="k8s_install_kubeadm" >}}
+    {{% tab name="Ubuntu, Debian or HypriotOS" %}}
+    # replace x in 1.13.x-00 with the latest patch version
+    apt-mark unhold kubeadm && \
+    apt-get update && apt-get install -y kubeadm=1.13.x-00 && \
```

Comment:
I don't think you really need to search and specify, it will default to latest for a series for both apt and yum

Reply:
me and @mauilion figured out that if the user delays the upgrade for multiple releases, they can end up upgrading to a minor version X+N, instead of X+1.